Organic/AmpAffiliateInjector.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Organic;
+
+use AMP_Base_Sanitizer;
```

Comment:
@sergunich This was the issue earlier--AMP_Base_Sanitizer wasn't included.

Reply:
Huh... yeah, this namespace resolution thing in PHP is a bit weird